neuralprophet/forecaster.py

```diff
@@ -2804,6 +2805,7 @@ def conformal_predict(
         alpha: Union[float, Tuple[float, float]],
         method: str = "naive",
         plotting_backend: Optional[str] = None,
+        show_qr: Optional[bool] = False,
```

Comment:
Recommend changing the naming of param `show_qr` to something else. Like show all PIs or show complete uncertainty or for advanced users etc. See comment below.